deblur/test/data/readme.txt

```diff
@@ -0,0 +1,7 @@
+70_otus.fasta - the 70% representative set from greengenes 13.5
+
+seqs_s1.fa - a simulated dataset with 100 reads derived from simset.s1.fasta using art
+seqs_s2.fa - a simulated dataset with 200 reads derived from simset.s2.fasta using grinder, with 10% chimeras, and additionally 35 phix reads
```

Comment:
is there still the more detailed explanation of how these were created (e.g. actual ART commands and expected sequences that were part of the comment in the code associated to these datasets)